quest_provider_plugins/usgs_ned.py

```diff
@@ -22,6 +30,12 @@ class UsgsNedServiceBase(SingleFileServiceBase):
         'elevation': 'elevation'
     }
 
+    def download(self, catalog_id, file_path, dataset, **kwargs):
```

Comment:
I do not know the context here, but you should be aware that Intake also has the ability to download source data files on first use https://intake.readthedocs.io/en/latest/catalog.html#caching-source-files-locally